package/src/components/MessageList/MessageList.tsx

```diff
@@ -1183,7 +1157,13 @@ const MessageListWithContext = <
       {!loading && (
         <>
           <View style={styles.stickyHeader}>
-            <StickyHeaderComponent />
+            {StickyHeader && (
+              <StickyHeader
+                date={stickyHeaderDate}
+                DateHeader={DateHeader}
+                messageListLengthAfterUpdate={messageListLengthAfterUpdate}
```

Comment:
```suggestion
                messageListLengthAfterUpdate={messageListLengthAfterUpdate}
```

I would not pass this as a prop, its something very specific forhe parent component.. and StickyHeader need not know about this

Reply:
basically what I understand is that if there is length show header, so we can do the same here 

messageListLengthAfterUpdate ? <<StickyHeader> : null

Reply:
Good suggestion. Thanks 😄